### PR TITLE
Write command to output stream when -vvv flag is set

### PR DESCRIPTION
**Pull request by jtgrimes**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
Documented?   | no
| Fixed tickets | #179

Per a discussion on issue #179, this PR outputs a copy of each task's command when the -vvv flag is used. I didn't update any docs, since I couldn't see where the -vvv flag is mentioned anywhere.

### CODE REVIEW

File: src/GrumPHP/Process/ProcessBuilder.php [hunk 1]
```diff
@@ -72,4 +80,14 @@ private function getCommandLocation($command)
...
         return $this->externalCommandLocator->locate($command);
     }
+
+    /**
+     * @param Process $process
+     */
+    private function writeVerboseOutput($process)
+    {
+        if ($this->io->isVeryVerbose()) {
+            $this->io->write(PHP_EOL.'Command: '.$process->getCommandLine(), true);
```
Comment (veewee) [inline src/GrumPHP/Process/ProcessBuilder.php]: Why did you add PHP_EOL? The last parameter of the method is already adding a newline.

Comment (jtgrimes) [inline src/GrumPHP/Process/ProcessBuilder.php]: The last arguments adds a newline at the end of the line, but the symfony ProgressBar class does't add a newline at the end of the task, so without it, we end up with ugly output like in the attached image.

grumphp

Another option would be to add the newline to the `$progressFormat` property of `Event\Subscirber\ProgressSubscriber` - in that case, we'd have a blank line between every task, which wasn't something I wanted to do.

**Comment by jtgrimes** [comment]
The spec should run correctly now - I develop on Windows and forgot that the command line arguments are escaped differently on Linux. The test should be platform independent now.

**Comment by veewee** [comment]
Thanks!
